### PR TITLE
Cypress test to check task parameters frame

### PR DESCRIPTION
**Pull request by dvkruchinin**
Cypress test to check if parameters "startFrame", "stopFrame", "frameStep" works as expected

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/support/commands.js [hunk 1]
```diff
@@ -30,12 +30,18 @@ Cypress.Commands.add('createAnnotationTask', (taksName='New annotation task',
...
-                                              additionalValue) => {
+                                              additionalValue,
+                                              advancedConfiguration=false,
+                                              multiJobs,
+                                              segmentSize,
+                                              sssFrame,
+                                              startFrame,
+                                              stopFrame,
+                                              frameStep
+                                              ) => {
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:44]: I would suggest to pass object here to this function uses available fields from the object

File: tests/cypress/integration/case_3_task_start_stop_step_frame.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                false, '', '', '', advancedConfiguration, multiJobs, '', sssFrame, startFrame, stopFrame, frameStep)
+            cy.openTaskJob(taskName)
+        })
+        it('Parameters "startFrame", "stopFrame", "frameStep" works as expected ', () => {
+            cy.get('.cvat-player-filename-wrapper')
+            .should('contain', `image_${labelName.replace(' ', '_').toLowerCase()}_${startFrame}.png`)
+            cy.get('.cvat-player-frame-selector').within(() => {
+                cy.get('input[role="spinbutton"]')
+                .should('have.value', '0')
+            })
```
Comment (bsekachev) [inline tests/cypress/integration/case_3_task_start_stop_step_frame.js:58]: This segment is repeated 3 times. Can we remove code duplication?

File: tests/cypress/support/commands.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js]: Thanks. Replaced. 
In 2 tests, I also had to slightly correct the task creation function.

File: tests/cypress/integration/case_3_task_start_stop_step_frame.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/case_3_task_start_stop_step_frame.js]: Code duplication removed.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**